engine/render/src/waf_render.py

```diff
@@ -37,8 +37,10 @@ def transform_material(task, msg):
 def font_file(self, node):
     classpath = [self.env['DYNAMO_HOME'] + '/share/java/bob-light.jar']
     fontmap = self.create_task('fontmap')
+
     fontmap.env['CLASSPATH'] = os.pathsep.join(classpath)
     fontmap.env['CONTENT_ROOT'] = fontmap.generator.content_root
+    fontmap.env['PROJECT_ROOT'] = fontmap.generator.project_root
```

Comment:
This feels a bit like a hack, but I don't want to break my back into fixing this the "right way" according to waf. I'd rather replace content building in waf with something simpler like a shell script..

Reply:
Well, it has less to do with Waf, and more to do with what path we want uin our tool chain.
Currently, the cases I've seen is that the content_path and project_path point to the same path, only with different formats (relative vs absolute).